pages/settings/wallets/nwc.js

```diff
@@ -47,10 +49,11 @@ export default function NWC () {
           initialValue={nwcUrl}
           label='connection'
           name='nwcUrl'
-          type='password'
+          type={showPass ? 'text' : 'password'}
           autoComplete='new-password'
           required
           autoFocus
+          append={<PasswordHider onClick={() => setShowPass(!showPass)} />}
```

Comment:
I think it makes sense to also pass in `showPass`. Then you don't need to keep any state inside `PasswordHider` and it would be less confusing than having two `setShowPass` functions which need to be synchronized.